app/services/food_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodService
+   def food_search(food)
+      response = conn.get("foods/search?query=#{food}&pageSize=10&pageNumber=1")
+      body = parsed(response)
+   end
+
+   private
+
+   def conn 
+      Faraday.new(url:'https://api.nal.usda.gov/fdc/v1/') do |faraday|
```

Comment:
Be careful about including part of the path in the faraday connection as that can sometimes cause some weird bugs. It's conventional to just add the base url (i.e. everything up to .gov) but I'd add `/fdc/v1/` to the path on line 3